### ISSUE TITLE
dbCreate is not working as expected

### ISSUE THREAD

**Issue by logangorence** [billysometimes/rethinkdb#25]
```dart
await r.dbCreate(dbName).run(conn).catchError((err) {
  print(err.message);
  print(err.frames);
});
```

Returns:
```
Expected 1 argument but found 0.
[]
```

**Comment by billysometimes** [comment]
Is dbName defined? If it is null then you would receive that error.

**Comment by logangorence** [comment]
I actually have a print right before it, that does prove it's not null, sorry I did not include that.

**Comment by billysometimes** [comment]
Would you mind posting your full code? I haven't been able to reproduce the issue

**Comment by logangorence** [comment]
Here's an example showing the problem(I can reproduce it here as well). https://github.com/logangorence/rethinkdb_issue25

**Comment by logangorence** [comment]
You won't be able to use my server, but it does show the same problem.

**Comment by billysometimes** [comment]
Thanks, that was a big help.  Fixed in 2.0.3+1!
